docarray/array/sqlite.py

```diff
@@ -6,5 +6,32 @@
 
 
 class DocumentArraySqlite(StorageMixins, DocumentArray):
+    """
+    DocumentArray that stores Documents in a `SQLite database <https://www.sqlite.org/index.html>`_.
```

Comment:
I would add a line to say that the Document are store on disk and not in memory and that it is the simpler way to deal with persistent da